### PR TITLE
Release v2.9.7+rev4 (WIP)

### PR DESCRIPTION
**Pull request by telphan**

### CODE REVIEW

File: layers/meta-resin-jetson/recipes-core/initrdscripts/files/resindataexpander [hunk 1]
```diff
@@ -23,7 +23,7 @@ resindataexpander_enabled() {
 
 resindataexpander_run() {
     info "resindataexpander: Expand data partition... "
-    parted -s /dev/$datadev -- resizepart 16 100%
+    parted -s /dev/$datadev -- resizepart 14 58724352s
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-core/initrdscripts/files/resindataexpander]: Partition index changed from 16 to 14. Why?

Comment (telphan) [reply layers/meta-resin-jetson/recipes-core/initrdscripts/files/resindataexpander]: We have removed two partitions as per: https://github.com/resin-os/resin-jetson-tx2/pull/40/commits/c863f7793d86d56d8d95d1ba74c99adf2dd4f890

File: layers/meta-resin-jetson/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+set -e
+
+path="/opt/tegra-binaries-signed"
+
+mtsbootpack="mce_mts_d15_prod_cr_sigheader.bin.encrypt"
+sc7="warmboot_wbheader.bin.encrypt"
+
+# Flash first-stage bootloader to reserved memory
+dd if="$path/$mtsbootpack" of=/dev/mmcblk0 bs=512 seek=58724353 count=8191
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: Do we need to use count here? This needs to change accordingly when these bins change so maybe just have it dd whatever size the binary has.
Unless there is a good reason to use count

Comment (telphan) [reply layers/meta-resin-jetson/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: The count is the size of the partition that used to exist there. If you go past the limit of the partition you will touch the next partition.

Comment (floion) [reply layers/meta-resin-jetson/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: The count argument to dd is used to denote the number of blocks to copy from input. Why would you want to have this hardcoded? I would assume that NVidia would always ship binaries which are the right size

Comment (floion) [reply layers/meta-resin-jetson/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: @telphan ?

Comment (telphan) [reply layers/meta-resin-jetson/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: @floion I see what you mean... You are right the count is completely unnecessary.

Comment (floion) [reply layers/meta-resin-jetson/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: Ok. The same thing is in the bootloader hooks if I recall correctly

Comment (telphan) [reply layers/meta-resin-jetson/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: Yes

Comment (floion) [reply layers/meta-resin-jetson/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board]: Did you change this yet @telphan ? I can still see count here. Wondering if it's how github shows this now?

**Pull request closed without merging**